hugegraph-api/src/main/java/com/baidu/hugegraph/api/traversers/AdamicAdarAPI.java

```diff
@@ -67,7 +67,7 @@ public String get(@Context GraphManager manager,
                       @QueryParam("max_degree")
                       @DefaultValue(DEFAULT_MAX_DEGREE) long maxDegree,
                       @QueryParam("limit")
-                      @DefaultValue(DEFAULT_ELEMENTS_LIMIT) long limit) {
+                      @DefaultValue(DEFAULT_ELEMENTS_LIMIT) int limit) {
```

Comment:
do all the algorithms(like adamicAdar) need to param limit with type int?

Reply:
Ok, I change the other algorithms limit type to int

Reply:
These place long are not suitable, because the java package only support int type on collections

Reply:
yes，I also found this problem~

Reply:
I have change all algorithms limit type long to int, and I have put a pr for hugegraph-client